github/actions_secrets.go

```diff
@@ -92,6 +113,11 @@ func (s *ActionsService) GetSecret(ctx context.Context, owner, repo, name string
 	return secret, resp, nil
 }
 
+// SelectedRepositoryIDs are the repository IDs that have access to the secret.
+type SelectedRepositoryIDs struct {
```

Comment:
Instead of making this a struct, how about if we make it a type so that it can be used more easily both within a response and as a request parameter?

```
type SelectedRepoIDs []int64
```